### PR TITLE
Replace seasonal flu with Zika in API examples

### PR DESCRIPTION
**Pull request by huddlej**
## Description of proposed changes

Replace seasonal flu with Zika in API examples based on [Slack discussion](https://bedfordlab.slack.com/archives/C03KWDET9/p1719943529502889?thread_ts=1719612970.204079&cid=C03KWDET9).

[Preview of changes to docs](https://nextstrain--941.org.readthedocs.build/projects/nextstrain-dot-org/en/941/api-restful.html#synopsis).

## Checklist

- [x] Checks pass
- [x] Check if changes affect the [resource index JSON revision](https://docs.nextstrain.org/projects/nextstrain-dot-org/en/latest/resource-collection.html#resource-index-revisions)

### CODE REVIEW

**Review by trvrb — APPROVED** [review]
Looks great!

File: docs/api-restful.rst [hunk 1]
```diff
@@ -33,16 +33,16 @@ Synopsis
 .. code-block:: bash
 
     # Download a main dataset JSON file…
-    curl https://nextstrain.org/seasonal-flu/h3n2/ha/2y                         \
+    curl https://nextstrain.org/zika                         \
```
Comment (victorlin) [inline docs/api-restful.rst:36]: Whitespace to align `\` across multiple lines is a bit off now. I'll fix in another PR

File: docs/api-restful.rst [hunk 2]
```diff
@@ -33,16 +33,16 @@ Synopsis
...
         --header 'Accept: application/vnd.nextstrain.dataset.main+json'         \
         --compressed                                                            \
-            > seasonal-flu_h3n2_ha_2y.json
+            > zika.json
```
Comment (victorlin) [inline docs/api-restful.rst:39]: I also contemplated replacing with another example that uses the `/` to `_` pattern, but realized that's not a feature relevant to this API and more for something like [`nextstrain remote download`](https://docs.nextstrain.org/projects/cli/en/8.5.0/commands/remote/download/) where that's clearly documented already.

File: docs/api-restful.rst (hunk 1 above)
Comment (victorlin) [reply docs/api-restful.rst:36]: https://github.com/nextstrain/nextstrain.org/pull/943

File: docs/api-restful.rst (hunk 2 above)
Comment (tsibley) [reply docs/api-restful.rst:39]: Hmm. I used seasonal-flu exactly for this reason: to highlight how multi-part pathogen URLs work and to show by example the local filesystem naming convention (even if you can ignore it, you're likely better off not). I'd suggest switching to mpox or ncov/open or something similar where we can maintain the multi-part example.

Comment (victorlin) [reply docs/api-restful.rst:39]: Done: https://github.com/nextstrain/nextstrain.org/pull/946

**Pull request merged**